src/components/menu/Menu.tsx

```diff
@@ -54,8 +55,8 @@ class Menu extends React.Component<WithTranslation, object> {
 
 
 interface IProps extends RouteComponentProps<any> {
-    to: string;
-    activeClassName: string;
+    to: H.LocationDescriptor | ((location: H.Location) => H.LocationDescriptor);
+    activeClassName?: string;
```

Comment:
What are those accomplishing? :thinking:

Reply:
That is a more accurate type for the `to` prop. I got it from the react router type definitions. 
I also made `activeClassName optional to add support for external links.

Both of these changes are no longer relevant to this PR, but I thought they can be useful in the future<sup>TM</sup>. Will revert them.